### ISSUE TITLE
How to check if session is valid with SSO

### ISSUE THREAD

**Issue by knowrick** [Clever/saml2#165]
We have a working version of the saml2-js library working with login, log out, and assertions.

But now we need to make sure that every request is valid against our SSO provider.

Example:

User Application Logins
User Application Asserts
Valid Group
Create a session
User Logins in
- Now we need to make sure every request is valid for this user
- If it's not valid kick them out of there session
- If it is valid. validate their session.
The user continues to use the application.

Here is an image of what I'm trying to set up now based on the CAS documentation.

![CAS Flow](https://www.saltyslopes.com/wp-content/uploads/2019/03/cas-flow.png)

In short, how can we validate every other request other than the initial one with the SSO?

Thanks,

Any help is much appreciated.
